rocAL/rocAL_pybind/amd/rocal/types.py

```diff
@@ -77,6 +77,11 @@
 from rocal_pybind.types import GAUSSIAN_INTERPOLATION
 from rocal_pybind.types import TRIANGULAR_INTERPOLATION
 
+#      Rocal Extrernal Source Mode
```

Comment:
Please remove additional space before Rocal Extrernal Source Mode
Please correct the spelling of comment too

Reply:
space is required. It is the format for types.py